test/trento/application/usecases/clusters_test.exs

```diff
@@ -64,6 +70,39 @@ defmodule Trento.ClustersTest do
 
       assert {:error, :amqp_error} = Clusters.request_checks_execution(cluster_id)
     end
+
+    test "should request cluster checks execution for all clusters" do
+      checks = [Faker.UUID.v4(), Faker.UUID.v4()]
+      %{id: cluster_id} = insert(:cluster, id: Faker.UUID.v4(), selected_checks: checks)
+      %{id: host_id1} = insert(:host, id: Faker.UUID.v4(), cluster_id: cluster_id)
+      %{id: host_id2} = insert(:host, id: Faker.UUID.v4(), cluster_id: cluster_id)
+
+      expect(Trento.Infrastructure.Messaging.Adapter.Mock, :publish, fn "executions", event ->
+        assert %ExecutionRequested{
```

Comment:
If you move this match directly inside the function signature I think it'll work the same

Reply:
Ah thanks, nice catch :D